catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -258,6 +243,96 @@ public void testMetacardTypeValuesCacheEnabled() throws Exception {
         assertReplies(replies);
     }
 
+    @Test
+    public void testReducedMaxQueryResults() throws Exception {
+
+        framework = createFramework(20);
```

Comment:
I meant setup in the unit test method itself. Looks like you went ahead and did that already with the `final int MAX_QUERY_RESULTS = 7;` and others. Thanks!